devTools/general/build-api-doc.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```

Comment:
may able to move this into the git action where it automatically publishes the doc

Reply:
This is a good idea. But since we didn't make a release with these changes we shouldn't publish development API docs. Isn't it? And we should have changelog for APIs, which API added which OHRM release.